.github/workflows/ubuntu.yml

```diff
@@ -42,14 +43,25 @@ jobs:
           libxcb-dri3-dev \
           libxext-dev \
           libxfixes-dev \
-          libwayland-dev
+          libwayland-dev \
+          meson
     - name: print compiler version
       run: ${{ matrix.compiler }} --version
+    - name: meson setup
```

Comment:
Would grouping meson and autotools together be a little better?
```
    # meson path
    - name: meson setup
      if: ${{ (matrix.build == 'meson') }}
      run: meson setup ./builddir --prefix=/usr
    - name: meson compile
      if: ${{ (matrix.build == 'meson') }}
      run: meson compile -C ./builddir || ninja -C ./builddir
    - name: meson install
      if: ${{ (matrix.build == 'meson') }}
      run: sudo meson install -C ./builddir

    # autotools path
    - name: configure
      if: ${{ (matrix.build == 'autotools') }}
      run: ./autogen.sh --prefix=/usr
    - name: build
      if: ${{ (matrix.build == 'autotools') }}
      run: make
    - name: install
      if: ${{ (matrix.build == 'autotools') }}
      run: sudo make install
```

Reply:
Thanks fixed